setup.py

```diff
@@ -21,8 +21,13 @@ def setup_package():
         classifiers=[
             "Development Status :: 5 - Production/Stable",
             "Programming Language :: Python",
+            'Programming Language :: Python :: 3',
+            'Programming Language :: Python :: 3.6',
```

Comment:
What about 3.7?

Reply:
The title says _minimal_ ;)

I'm currently setting up a proper build matrix. I'll update this once it's done